backlog-set-due-date

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -e
+# Usage using a file for testing purposes: backlog-set-due-date <file>
+
+if [ $# -eq 1 ]; then
+  file=$1
+  if [ ! -f "$1" ]; then
+    echo "Usage: $0 <valid issues json-file>"
+    exit 2
+  fi
+fi
+
+if [ -z "$file" ]; then
+  redmine_api_key="${redmine_api_key:?"Need redmine API key"}"
+fi
+
+project_id="${project_id:-18}"
+ticket_limit="${ticket_limit:-200}"
+host="${host:-"https://progress.opensuse.org"}"
+issues=$(mktemp)
+
+if [ -z "$file" ]; then
+  curl -s -H "X-Redmine-API-Key: $redmine_api_key" "$host/issues.json?project_id=$project_id&limit=$ticket_limit" | jq -r \
+  '.issues | .[] |  select(.priority.name!="Low" and .due_date==null and .assignee==null)' >"$issues"
+else
+  jq -r '.issues | .[] |  select(.priority.name!="Low" and .due_date==null and .assignee==null)' "$1" >"$issues"
+fi
+
+for id in $(jq .id "$issues"); do
+  due_date=$(date -d"+14 days" +%Y-%m-%d)
+  if [ -z "$file" ]; then
```

Comment:
How about `dry_run="${dry_run:-"0"}"` instead like is done in other scripts?

Reply:
I commit this idea for review